smartrack.rb

```diff
@@ -22,7 +23,7 @@
 
   sleep 1
   if i == 0 then
-    if @driver.find_element(:css, 'input#number_of_tests').displayed? then
+    if @driver.find_element(:css, 'input#ctl00_MainContent_txtUsername').displayed? then
```

Comment:
Line is too long. [88/80]<br>Do not use `then` for multi-line `if`.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.